### ISSUE TITLE
Resource Not Found - The requested resource could not be found.

### ISSUE THREAD

**Issue by hiteshkoli** [thinkshout/mailchimp-api-php#97]
Hi ,

I am using the library along with `Mailchimp E-Commerce` for Drupal. However,  I am getting `Resource Not Found - The requested resource could not be found.` when I add a product to the cart. I have enabled option to send shopping cart information to Mailchimp everytime a product is added to the cart. The only response I get is `Resource Not Found - The requested resource could not be found.`

Anyone, can shed lights on what possibly can be the issue ? 

Details are here : https://www.drupal.org/project/mailchimp_ecommerce/issues/3134642#comment-13608937

**Comment by ruscoe** [comment]
Total guess here, but did you sync your store's products to Mailchimp?

If not, there's an option to sync products at `/admin/config/services/mailchimp/ecommerce`

I suspect the module is sending cart information containing products that Mailchimp isn't aware of. Syncing should fix that.

**Comment by hiteshkoli** [comment]
@ruscoe , yeah I had synced the products to Mailchimp before creating orders. 

What surprises me is that the order is send to Mailchimp when I uncheck the option (send shopping cart information). When this option is unchecked the shopping cart information is sent to Mailchimp at the final checkout (purchase) rather than sending status of the cart everytime. 

However, when it is checked the shopping cart information is not sent like adding product or updating

![Screen Shot 2020-05-15 at 9 45 54 am](https://user-images.githubusercontent.com/1768337/81996720-13830780-9691-11ea-9cd2-e031c1b73a24.png)
 cart.
